### PR TITLE
added token decimals and token name to chainspec

### PR DESCRIPTION
**Pull request by JesseAbram**
Hey, I was looking through your blockchain, looks great saw some things I just did for the substrate blockchain I was building and thought Id contribute. I added a token name and decimal (guessed the values in the config file). There is a bunch more I can do by the way, like set up telemetry in the chainspec, by default make nodes spin up off a chianspec.....which due to WASM's non deterministic compilation would make it hard to connect separate nodes not on the same machine if not done of a pre compiled chainspec.

### CODE REVIEW

**Comment by adamdossa** [comment]
@JesseAbram thanks for taking a look!

Are these properties (name, decimals) used by polkadot.js?

**Comment by JesseAbram** [comment]
No, So the UI for polkadot defaults to 18 decimal points.....I am not sure about the token name but I assume it would also be dot. However the parachains like Robonomics, Edgeware, the one Im working on do and it is reflected in the UI. https://github.com/airalab/substrate-node-robonomics/blob/master/bin/node/cli/src/chain_spec.rs#L276 for example 


and in the chainspec you see 

image

File: src/chain_spec.rs [hunk 1]
```diff
@@ -158,6 +160,13 @@ impl Alternative {
     }
 }
 
+fn polymath_props() -> Properties {
+    json!({"tokenDecimals": 18, "tokenSymbol": "TEST" })
```
Comment (maxsam4) [inline src/chain_spec.rs]:
```suggestion
    json!({"tokenDecimals": 6, "tokenSymbol": "POLY" })
```

File: src/chain_spec.rs [hunk 2]
```diff
@@ -157,6 +159,13 @@ impl Alternative {
     }
 }
 
+fn polymath_props() -> Properties {
+    json!({"tokenDecimals": 6, "tokenSymbol": "POLY" })
```
Comment (fmiguelgarcia) [inline src/chain_spec.rs]: @JesseAbram We are avoiding the use of any code that could generate `panic` (safety reason), like `unwrap`.
This is not the case, but it is going to be easier to find `panic` functions if we limit their use.
In this PR, you could just use something like: 
```
fn polymath_props() -> Option<Properties> {
    json!({"tokenDecimals": 6, "tokenSymbol": "POLY" }).as_object().cloned()
}
```

Comment (JesseAbram) [reply src/chain_spec.rs]: Just for my own personal understanding. 

unwrap would handle a null value as a panic, which is against your coding standards...... which ya running a node makes sense. However, in this scenario the data would be hardcoded and would be compiled pre running the node into a chianspec and not able to be called again. 

Totally get why the change was made as coding standards, just personally trying to get a better understanding.

Comment (fmiguelgarcia) [reply src/chain_spec.rs]: You're right this code is **completely safe**. :+1: 
It is just a facility to keep any `unwrap` (or any other `panic` function) at minimum. We could use a simple `grep` to count the number of `unwrap` and add this as a kind of "safety KPI".
We are using `unwrap` only in _unit test_, and removing them at **production code**.

Less `panic` fn (or `unsafe` fn) means safer code :smile:

Comment (JesseAbram) [reply src/chain_spec.rs]: Makes sense, Thank you!

**Pull request merged**